accessibility/test.js

```diff
@@ -1,6 +1,6 @@
 const AxeBuilder = require('@axe-core/webdriverjs');
 const WebDriver = require('selenium-webdriver'),
-      By = WebDriver.By;
+    By = WebDriver.By;
 const firefox = require('selenium-webdriver/firefox');
 
 const username = `${process.env.TEST_USERNAME}`
```

Comment:
Should `/account/logout/` be added to the `pagesToAnalyze` array in this file?

Reply:
Completely agree - that's been tested and added

Reply:
Everything else looks good from me